tests/bgp/test_bgp_4-byte_as_trans.py

```diff
@@ -0,0 +1,290 @@
+             setup['neigh2_ip_v6'], setup['neigh2'], setup['neigh2_ip_v6'], setup['neigh2_ip_v6'],
+             setup['dut_ipv4_network'], setup['peer_group_v4'], setup['peer_group_v4'], setup['peer_group_v4'],
+             setup['neigh_ip_v4'], setup['neigh2_ip_v4'], setup['dut_ipv6_network'], setup['peer_group_v6'],
+             setup['peer_group_v6'], setup['peer_group_v6'], setup['neigh_ip_v6'], setup['neigh2_ip_v6'])
+    logger.debug(setup['duthost'].shell(cmd, module_ignore_errors=True))
+
+    cmd = 'vtysh \
+    -c "config" \
+    -c "no router bgp {}" \
+    -c "router bgp {}" \
+    -c "bgp router-id {}" \
+    -c "bgp log-neighbor-changes" \
+    -c "no bgp ebgp-requires-policy" \
+    -c "no bgp default ipv4-unicast" \
+    -c "bgp bestpath as-path multipath-relax" \
+    -c "neighbor {} peer-group" \
+    -c "neighbor {} peer-group" \
+    -c "neighbor {} remote-as {}" \
+    -c "neighbor {} peer-group {}" \
+    -c "neighbor {} description {}" \
+    -c "neighbor {} timers 3 10" \
+    -c "neighbor {} timers connect 10" \
+    -c "neighbor {} remote-as {}" \
+    -c "neighbor {} peer-group {}" \
+    -c "neighbor {} description {}" \
+    -c "neighbor {} timers 3 10" \
+    -c "neighbor {} timers connect 10" \
+    -c "address-family ipv4 unicast" \
+    -c "network {}" \
+    -c "neighbor {} soft-reconfiguration inbound" \
+    -c "neighbor {} route-map FROM_BGP_PEER_V4 in" \
+    -c "neighbor {} route-map TO_BGP_PEER_V4 out" \
+    -c "neighbor {} activate" \
+    -c "maximum-paths 64" \
+    -c "exit-address-family" \
+    -c "address-family ipv6 unicast" \
+    -c "network {}" \
+    -c "neighbor {} soft-reconfiguration inbound" \
+    -c "neighbor {} route-map FROM_BGP_PEER_V6 in" \
+    -c "neighbor {} route-map TO_BGP_PEER_V6 out" \
+    -c "neighbor {} activate" \
+    -c "maximum-paths 64" \
+    -c "exit-address-family" \
+    '.format(setup['neigh_asn'], neighbor_4byte_asn, setup['neigh_bgp_id'],
+             setup['peer_group_v4'], setup['peer_group_v6'], setup['dut_ip_v4'], dut_4byte_asn, setup['dut_ip_v4'],
+             setup['peer_group_v4'], setup['dut_ip_v4'], 'DUT', setup['dut_ip_v4'], setup['dut_ip_v4'],
+             setup['dut_ip_v6'], dut_4byte_asn, setup['dut_ip_v6'], setup['peer_group_v6'], setup['dut_ip_v6'], 'DUT',
+             setup['dut_ip_v6'], setup['dut_ip_v6'], setup['neigh_ipv4_network'], setup['peer_group_v4'],
+             setup['peer_group_v4'], setup['peer_group_v4'], setup['dut_ip_v4'], setup['neigh_ipv6_network'],
+             setup['peer_group_v6'], setup['peer_group_v6'], setup['peer_group_v6'], setup['dut_ip_v6'])
+
+    logger.debug(setup['neighhost'].shell(cmd, module_ignore_errors=True))
+
+    logger.debug("DUT BGP Config: {}".format(setup['duthost'].shell("show run bgp")['stdout']))
+    logger.debug("Neighbor BGP Config: {}".format(setup['neighhost'].shell("show run bgp")['stdout']))
+
+    time.sleep(bgp_sleep)
```

Comment:
could you elaborate on what the previous cmd is doing?
the sleep time could be different among T2s

Reply:
The previous command configures a 4-byte ASN on the neighbor device.  
I'm not sure what you mean by the sleep time being different as this is a variable set in the script.

Reply:
I mean how did you decide bgp_sleep?

Reply:
It is the standard amount of time we wait for our tests to allow BGP to converge.  Then after running the test case if there is an issue it is extended.

Reply:
LGTM if this sleep time of 120 works well on T2